pyproject.toml

```diff
@@ -174,13 +174,14 @@ filterwarnings = [
 addopts = "-ra"
 
 [tool.cibuildwheel]
-build = "cp311-* cp312-* cp313-*"
+enable = ["cpython-prerelease"]
```

Comment:
this is impractical and needs to be removed from this PR before merging:
- to avoid building numpy in tests, I need to set up a `uv.sources` table
- doing so invalidates the lock file and will lock *all* CI on dev versions of multiple deps (not sure if this is desired or a uv bug, but I'm not interested in finding out)